packages/mdc-textfield/README.md

```diff
@@ -363,17 +361,12 @@ Activates the focus state of the Text Field. Normally called in response to the
 
 Deactivates the focus state of the Text Field. Normally called in response to the input blur event.
 
-##### MDCTextFieldFoundation.animateBottomLine(evt: Event)
-
-Animates the bottom line. The animation expands outward from the user's click or touch. Expects an
-event with clientX/clientY properties.
-
 ##### MDCTextFieldFoundation.handleBottomLineAnimationEnd(evt: Event)
 
 Handles the end of the bottom line animation, performing actions that must wait for animations to
 finish. Expects a transition-end event.
 
-##### MDCTextField.helperTextContent
+##### MDCTextField.setHelperTextContent(content)
```

Comment:
Should this be `MDCTextFieldFoundation.setHelperTextContent`, given that this is in the foundation API section? (The foundation has `setHelperTextContent`; the component still has a `helperTextContent` setter.)

Reply:
Good catch, thanks!